### DISCUSSION TITLE
Refresh The Resource From A Resource Widget

### DISCUSSION THREAD

**Discussion by Milkhan** [filamentphp/filament#2608] [Help] [answered]
Hi, first of all, thank you so much, working on the filament is super nice!

I have created a widget and i am inserting bulk records from an input in the widget. My questions is how can i refresh the resource table and notify the user about the results?

```
\Filament\Facades\Filament::notify('success', 'Records added successfully', isAfterRedirect: true);
        
return redirect()->to('/admin/school-classes');
```

When I use isAfterRedirect and redirect to the same resource page to see the latest records, the flush message is not working.

**Comment by danharrin** [comment, marked as answer]
`isAfterRedirect` is deprecated btw, you don't need to use it any more

Instead of refreshing the page through redirect, emit a [Livewire event](https://laravel-livewire.com/docs/2.x/events#from-component) from the widget and [listen](https://laravel-livewire.com/docs/2.x/events#event-listeners) for it on the List page class, calling the `$refresh` action on the listener.

**Reply by Milkhan** [reply]
@danharrin Thank you so much!
